src/jvm/main/org/jetbrains/kotlinx/lincheck/CTestStructure.java

```diff
@@ -210,6 +264,11 @@ private static ParameterGenerator<?> createGenerator(Param paramAnn, RandomProvi
         }
     }
 
+    @SuppressWarnings({"rawtypes", "unchecked"})
```

Comment:
Just "@Suppress"?

Reply:
Complier doesn't allow me to write `@Suppress`, it forces me to write `@Suppress(names = {...})`

Reply:
`@Suppress("unchecked", "rawtypes")`?

Reply:
As we're in Java file, this approach doesn't work, only `@Suppress(names = {"unchecked", "rawtypes"})` works, but it suppresses nothing, so I suggest to keep current version.